src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -748,15 +759,21 @@ private protected Compilation RunGenerators(Compilation input, ParseOptions pars
                                                   .ReplaceAdditionalTexts(additionalTexts);
             }
 
-            driver ??= CreateGeneratorDriver(parseOptions, generators, analyzerConfigOptionsProvider, additionalTexts);
+            var driverOptions = new GeneratorDriverOptions(IncrementalGeneratorOutputKind.None, trackIncrementalGeneratorSteps);
```

Comment:
Consider inlining this statement into the next (no need to do this work when `driver` isn't null.